uportal-war/src/main/java/org/jasig/portal/groups/pags/dao/jpa/PersonAttributesGroupTestDefinitionImpl.java

```diff
@@ -46,14 +51,14 @@
  * @author Shawn Connolly, sconnolly@unicon.net
  */
 @Entity
-@Table(name = "UP_PAGS_TEST")
+@Table(name = JpaPersonAttributesGroupTestDefinitionDao.TABLENAME_PREFIX)
```

Comment:
:+1:  I'm all for that, I just wasn't into that many files for this enhancement.  (And I didn't want to add several files to the delta for something so small.)  I support this change, FWIW.